### DISCUSSION TITLE
Fail to compile after last update

### DISCUSSION THREAD

**Discussion by Alaa-Aim** [vortigont/espem#35] [Q&A]
@vortigont , Hi, after last update for this project, I couldn't compile, build log error as below :

![image](https://user-images.githubusercontent.com/64470118/235370623-a0a9631c-3d79-4812-855a-6cedc37eb87c.png)

![image](https://user-images.githubusercontent.com/64470118/235370650-ce991329-179c-47de-862d-fb2aed043cc8.png)

![image](https://user-images.githubusercontent.com/64470118/235370688-ed0df280-92d9-4abb-94e6-79a75000fb6e.png)

**Comment by vortigont** [comment]
It builds find for me. Pls, try to purge .pio folder.

**Reply by Alaa-Aim** [reply]
@vortigont , I did removed all .pio folder and still have the same exact issue, I noticed the library manger still download FTPClientServer library 

![image](https://user-images.githubusercontent.com/64470118/235465972-e691ab03-7244-4da3-8ff2-9df3024fa1b9.png)

**Comment by vortigont** [comment]
Looks like esp32 platform has been updated recently and it introduced some compatibility issues.
Not sure what is that with FTP lib,  but it could be disabled with `EMBUI_NOFTP` flag.
Let me check what are those new issues, I'll try to fix it

**Comment by vortigont** [comment]
should be OK now, I've forked an FTP lib and patched it. Will file a PR to upstream

**Reply by Alaa-Aim** [reply]
@vortigont Hi, worked like a charm, many thanks
